db/migrate/20180719125034_remove_admin_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAdminFromUsers < ActiveRecord::Migration[5.1]
+  def change
+    remove_column :users, :admin
```

Comment:
Always try to mention `data type` in `remove_column`. As it useful in migeration rollback